schedule/yast/yast2_ncurses/yast2_ncurses_gnome.yaml

```diff
@@ -29,18 +29,20 @@ conditional_schedule:
         - console/yast2_lan_hostname
         - console/yast2_settings
         - console/yast2_dns_server
+        - console/yast2_nfs_server
         - console/yast2_nfs_client
         - console/yast2_snapper_ncurses
       aarch64:
         - console/yast2_lan
+        - console/yast2_nfs_server
       ppc64le:
         - console/yast2_lan
+        - console/yast2_nfs_server
       s390x:
         - console/yast2_rmt
         - console/yast2_ntpclient
         - console/yast2_tftp
         - console/yast2_lan
         - console/yast2_lan_hostname
         - console/yast2_dns_server
-        - console/yast2_nfs_client
```

Comment:
did you tried to add the server and removed the client accidentally? [that one](https://openqa.suse.de/tests/4694250#step/yast2_nfs_client/20) is working.